docs/tutorials/first-post-type.md

```diff
@@ -1,36 +1,140 @@
 # Creating Your First Post Type
 
-A step-by-step guide to creating a custom post type using Secure Custom Fields.
+A step-by-step guide to creating a custom post type using Secure Custom Fields (SCF).
+
+**What is a custom post type?**
+
+In WordPress, a custom post type (CPT) is a content type like posts and pages, but customized to suit your specific needs. You can use CPTs to manage products, portfolios, testimonials, events, vehicles, and more—essentially any type of structured content that needs its own menu and fields.
 
 ## Prerequisites
 
-- SCF installed and activated
-- Administrator access to WordPress
-- Basic understanding of WordPress concepts
+-   SCF installed and activated
+-   Administrator access to WordPress
+-   Basic understanding of WordPress concepts
+
+You can learn more about WordPress basics here:
+
+-   [Theme Basics](https://developer.wordpress.org/themes/basics/)
+-   [Plugin Basics](https://developer.wordpress.org/plugins/plugin-basics/)
+
+## 1. Access the Admin Panel
+
+To begin creating your custom post type:
+
+-   Go to **Secure Custom Fields → Post Types** in your WordPress admin menu.
+-   Click the **"Add New"** button to open the creation form.
+
+## 2. Basic Configuration
+
+These fields help you define how your post type will appear and behave:
+
+### Plural Label \*
+
+This is the name that will appear in the admin sidebar. For example, `Movies`.
+
+### Singular Label \*
+
+Used for individual entries. Example: `Movie`.
+
+### Post Type Key \*
+
+A technical identifier for WordPress to recognize this post type. Use lowercase letters and underscores only. Max 20 characters. Example: `movie`.
+
+### Taxonomies
+
+Choose existing taxonomies like **category** or **tags** if you want to group or filter your content. You can also select custom ones like **Brand** or **Color**.
+
+### Public
+
+Choose **Yes** to make your post type visible on the website and admin area. Choose **No** to keep it private.
+
+### Hierarchical
+
+Set to **Yes** if you want your items to be nested (like pages). Set to **No** for a flat list (like blog posts).
+
+### Advanced Configuration
+
+Enable this to unlock more advanced options, useful when you need more control over how your post type works.
+
+## 3. Advanced Settings
+
+These options let you customize deeper behaviors and labels.
 
-## Steps
+### Supports
 
-1. **Access the Admin Panel**
-   - Navigate to Custom Fields → Post Types
-   - Click "Add New"
+Select which features to enable when editing a post, such as:
 
-2. **Basic Configuration**
-   - Enter a descriptive name
-   - Configure labels
-   - Set visibility options
+-   **Title**: Adds a title field
```

Comment:
I just noticed the spaces _after_ the hyphen; is there any linting rule that requires this?